apps/engine/lib/execution/src/package/simulation/init/task.rs

```diff
@@ -8,24 +8,28 @@ use crate::{
     Error, Result,
 };
 
+
 /// All init package tasks are registered in this enum
 #[derive(Clone, Debug)]
 pub enum InitTask {
     JsInitTask(JsInitTask),
+    TsInitTask(TsInitTask),
     PyInitTask(PyInitTask),
 }
 
 impl Task for InitTask {
```

Comment:
Overall I think the traits for `Task` could do with some overhaul, but one thing in particular I've considered is applying [this technique](https://github.com/hashintel/hash/pull/2653) to `InitTask`.